includes/admin/payments/orders.php

```diff
@@ -742,32 +794,55 @@ function edd_order_details_extras( $order = false ) {
 			<div class="edd-admin-box">
 				<?php do_action( 'edd_view_order_details_payment_meta_before', $order->id ); ?>
 
-				<?php if ( ! edd_is_add_order_page() && ! empty( $order->id ) ) : ?>
-					<?php if ( ! empty( $order->gateway ) ) : ?>
+
+				<?php if ( ! edd_is_add_order_page() ) : ?>
```

Comment:
I'm comparing the behavior of the gateway and IP address fields on the existing orders and they're currently different, but I think they should be the same. However, I'm not sure of the best way to make them the same.
- the gateway field is no longer an input group, etc. at all, and the `<label>` has been removed
- the IP field still has a `<label>` element, but it's orphaned because an `<input>` is not added if it's an existing order

I think the gateway method makes more sense--partly because the IP address includes a link, so it can't be in a text input. Could we maybe define markup for a situation like this, and keep them in an `edd-form-group` container still, but then have inner `<span>` elements (or `<div>`, if we're adding CSS anyway) with `edd-` classes? Y'all will know better than I, but something like:

```html
<div class="edd-form-group">
	<span class="edd-form-group__label--readonly">Gateway</span>
	<span class="edd-form-group__input--readonly">Test Payment</span>
</div>
```
@spencerfinnell @LisaCee

Reply:
@robincornett What about making a separate component that utilizes `dl` (and `dt` and `dd`) for these key + value read-only outputs? I'm not sure the `form` nomenclature is necessary for plain text output. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

Reply:
Sure, we could try that--if something semantic exists which makes more potential sense than just generic `<span>` and `<div>` then I'm willing to try it.

@spencerfinnell are there order items, whether on the add or edit screen, which should not be editable? The IP address appears to be something which should never be edited, but would that have to stay in an input field on the add screen so that it would be saved to the order meta? On existing orders, the gateway and IP shouldn't be edited, but I'm thinking maybe the date also should not be changed (unless it was a manual payment in the first place, maybe)?

Reply:
@robincornett _What_ and _why_ things are editable is a lengthy discussion. The current items need to stay as-is. Unfortunately there isn't a discussion I can link to, but it has been decided previously to allow this.

Reply:
@robincornett For the sake of keeping this PR moving forward and focused on `.edd-form-group` only, I think we should address creating a generic `edd-dl` at a later date.

@LisaCee For IP, when viewing an Order let's ensure we aren't using an actual `label` element for the label, and that the items align to the left and right -- so it's setup just like Gateway on both Add and Edit views.

Reply:
Like this?
![image](https://user-images.githubusercontent.com/18077352/80243218-a644ff00-861b-11ea-9ca1-bc4187d99e37.png)

Reply:
@LisaCee Exactly

Reply:
I don't know why I would ever need the key, but it bothers me a bit that it's not wide enough to show the entire thing?

And I think the Gateway/IP look great that way, thank you @LisaCee!

Reply:
And you want it to look like that in add?  Right now I'm getting this: 
![image](https://user-images.githubusercontent.com/18077352/80243737-95e15400-861c-11ea-87a9-e46606ea263b.png)

Do you want the gateway select to be right aligned?

Reply:
@LisaCee It should still be a normal input when Adding an Order, so the IP can be set to something specific, other than the site admin's IP.

Reply:
The gateway select should not be right aligned--it should look just like a normal select field. I mean, as much as it can. Would love to update our Chosen styles to look a little more WP-ish but that's a different issue.

Reply:
Add

Gateway: editable `<select>` with label (standard `edd-form-control`)
IP: editable `<input>` with label (standard `edd-form-control`)

Edit/View

Gateway: right aligned static text
IP: right aligned static text

Reply:
Alignment-wise for Add an Order?
![image](https://user-images.githubusercontent.com/18077352/80245668-f0c87a80-861f-11ea-8664-628e76e1143f.png)